aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -389,6 +389,18 @@ def mangle_db_name(self, db_name: str) -> str:
 
 class PGSource(PGCluster):
     """Source PostgreSQL cluster"""
+    def get_size(self, *, dbname, only_tables: Optional[List[str]]) -> float:
+        if only_tables == []:
+            return 0
+        if only_tables is not None:
+            query = "SELECT SUM(pg_total_relation_size(tablename)) AS size FROM UNNEST(%s) AS tablename"
```

Comment:
For this query, the `dbname` should be passed to `self.c`, because `pg_total_relation_size` takes the table name and looks for this table in the database of the current connection.

It would be good to add specific tests for that function.

Reply:
You're right. I'll add some unit tests there as well.